declarations.d.ts

```diff
@@ -10,3 +10,5 @@ declare module "*.png" {
 
   export = value;
 }
+
+declare module "nft.storage/dist/bundle.esm.min.js";
```

Comment:
why do you need this?

Reply:
Webpack has some issues while importing nft.storage
https://github.com/nftstorage/nft.storage/issues/876#issuecomment-985687135

`import { NFTStorage, File } from "nft.storage/dist/bundle.esm.min.js";
`
For this import we have to declare it on declaration file.

Reply:
`packages/candymachine/nft-storage-upload.ts` removed since we're using `packages/candymachine/pinata-upload.ts`